src/pages/workspaces/workspace/Data.js

```diff
@@ -601,6 +612,13 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
         workspace,
         selectedEntities: _.keys(selectedEntities), selectedDataType: entityKey, runningSubmissionsCount
       }),
+      viewData && h(Modal, {
```

Comment:
Nit: coerce `viewData` to a boolean.

General question for others: this is a habit I picked up from this project when I came across `!!` for the first time. Is this generally a good thing to do any time a non-boolean is being used in a boolean context? Is there any case where that's not a good idea?

Reply:
Yes, it's generally nice because it communicates intent, and can help readers differentiate between e.g. boolean logic vs. default/override logic.